javascript/features/death_match/death_match_commands.js

```diff
@@ -21,9 +23,19 @@ export class DeathMatchCommands {
     }
 
     // The death match command is being used.
-    onDmCommand(player, zone) { 
-        if(zone === null || zone === undefined || typeof zone !== 'number') {
-            player.sendMessage(Message.DEATH_MATCH_INVALID_ZONE, zone);
+    async onDmCommand(player, zone) { 
+        if(isNaN(zone)) {
+            const dialog = new Menu('Choose a death match zone.', ['Zone', 'Name', 'Teams', 'Lag shot']);
+            for(const zoneId of this.manager_.validDmZones()) {
+                const location = DeathMatchLocation.getById(zoneId);  
+                dialog.addItem(location.id, location.name, location.hasTeams, location.lagShot, 
```

Comment:
What does this look like? Do we display "true" and "false" in the menu now?

Reply:
Yes, I assumed this is understandable by the user?
I thught about adding 1/0 but figured that wouldn't imrpove that much.

Maybe a Yes/No would be better if thought not to be understandable by the user?

Reply:
Discussed with player: "Yes/No" is preferred